workitem/workitem.go

```diff
@@ -123,13 +136,26 @@ func (wi WorkItem) ChangeSet(older convert.ChangeDetector) ([]convert.Change, er
 		})
 		return changes, nil
 	}
-	thisCopy := make([]string, len(bcThis))
-	olderCopy := make([]string, len(bcOlder))
-	copy(thisCopy, bcThis)
-	copy(olderCopy, bcOlder)
-	sort.Strings(thisCopy)
-	sort.Strings(olderCopy)
-	if !reflect.DeepEqual(thisCopy, olderCopy) {
+	// because go has a fundamentally broken handling of interface{}
```

Comment:
Just say what you're doing here would be more productive.

Reply:
At least I am adding comments.